workloads/api_load/common.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+source env.sh
+source ../../utils/common.sh
+source ../../utils/benchmark-operator.sh
+
+openshift_login
+
+export_defaults() {
+  network_type=$(oc get network cluster -o jsonpath='{.status.networkType}' | tr '[:upper:]' '[:lower:]')
+  export UUID=$(uuidgen)
+  export CLUSTER_NAME=$(oc get infrastructure cluster -o jsonpath="{.status.infrastructureName}")
+  local baremetalCheck=$(oc get infrastructure cluster -o json | jq .spec.platformSpec.type)
+  zones=($(oc get nodes -l node-role.kubernetes.io/workload!=,node-role.kubernetes.io/infra!=,node-role.kubernetes.io/worker -o go-template='{{ range .items }}{{ index .metadata.labels "topology.kubernetes.io/zone" }}{{ "\n" }}{{ end }}' | uniq))
+  platform=$(oc get infrastructure cluster -o jsonpath='{.status.platformStatus.type}' | tr '[:upper:]' '[:lower:]')
+  #Check to see if the infrastructure type is baremetal to adjust script as necessary 
+  if [[ "${baremetalCheck}" == '"BareMetal"' ]]; then
+    log "BareMetal infastructure: setting isBareMetal accordingly"
+    isBareMetal=true
+  else
+    isBareMetal=false
+  fi
+
+  if [[ "${isBareMetal}" == "true" ]]; then
+    #Installing python3.8
+    sudo yum -y install python3.8
+  fi
```

Comment:
Do you really need this?

Reply:
I mean will we ever run this script from baremetal? I'm OK leaving it around :)

Reply:
I will remove this (for code simplicity) as out team may not be using baremetal OCP deployment.